docs/patterns/components/ProgressBar/index.js

```diff
@@ -42,79 +42,54 @@ const ProgressBarDemo = () => {
-          '<ProgressBar aria-label="Progress" progress={5} progressMin={1} progressMax={25} />'
-        }
-      </Code>
-      <h3>Animated</h3>
-      <p>
-        If the progress indicates that some activity is currently progressing,
-        the progress bar will update the status of the progress bar based on the
-        current <code>progress</code> prop:
-      </p>
-      <p>
-        <Button onClick={handleAnimate}>Show Animated Progress</Button>
-      </p>
-      <ProgressBar aria-label="Progress" progress={progress} />
-      <h2>Code Sample</h2>
-      <Code language="javascript" role="region" tabIndex={0}>
-        {`import React from 'react';
-import { ProgressBar } from '@deque/cauldron-react';
-
-function Component() {
   return (
     <div>
-      <ProgressBar label="Current Progress" aria-describedby="progress" progress={75} />
-      <span id="progress">3 of 4 items complete</span>
-    </div>
-  )
-}`}
-      </Code>
-      <h2>Props</h2>
-      <div className="Demo-props">
-        <PropDocs
-          docs={{
-            progress: {
-              type: 'number',
-              required: true,
-              description: 'The current progress bar progress'
-            },
-            progressMin: {
-              type: 'number',
-              defaultValue: '0',
-              description: 'Minimum value of progress'
-            },
-            progressMax: {
-              type: 'number',
-              defaultValue: 100,
-              description: 'Maximum value of progress'
-            }
-          }}
-          defaultProps={{}}
-        />
-      </div>
+      <Demo
+        component={ProgressBar}
+        componentDescription={
+          'Progress bars are used to indicate the status of some progress activity happening, such as number of items completed.'
+        }
+        states={[
+          { 'arial-label': 'Progress', progress: 75 },
+          {
+            'aria-label': 'Progress',
+            progress: 5,
+            progressMin: 1,
+            progressMax: 25
+          }
+        ]}
+        propDocs={{
+          progress: {
+            type: 'number',
+            required: true,
+            description: 'The current progress bar progress'
+          },
+          progressMin: {
+            type: 'number',
+            defaultValue: '0',
+            description: 'Minimum value of progress'
+          },
+          progressMax: {
+            type: 'number',
+            defaultValue: 100,
+            description: 'Maximum value of progress'
+          }
+        }}
+      >
+        <h2>Animated</h2>
+        <p>
+          If the progress indicates that some activity is currently progressing,
+          the progress bar will update the status of the progress bar based on
+          the current <code>progress</code> prop:
+        </p>
+        <Code language="javascript" role="region" tabIndex={0}>
+          {`<ProgressBar aria-label="Progress" progress={progress} />`}
+        </Code>
```

Comment:
Can we move this code snippet to the bottom of the example similar to how the other ones are displayed above?

<img width="1177" alt="Screen Shot 2022-10-11 at 10 29 44 AM" src="https://user-images.githubusercontent.com/3081483/195148386-29687bd6-8b22-4bb2-8fb0-c581ab031b7a.png">

Reply:
done!